lib/win32/certstore.rb

```diff
@@ -54,6 +54,12 @@ def add(cert_file_path)
       return add
     end
 
+    def delete(certificate_name)
+      delete_cert = cert_delete(@certstore_handler, certificate_name)

```

Comment:
when we have an `attr_reader` we can just call `certstore_handler` instead of `@certstore_handler`. This allows a future refactor where we could replace the reader with a method if we wanted to do something else.